client/src/cmdhfmf.c

```diff
@@ -2570,11 +2573,13 @@ static int CmdHF14AMfAutoPWN(const char *Cmd) {
 static int CmdHF14AMfChk_fast(const char *Cmd) {
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "hf mf fchk",
-                  "This is a improved checkkeys method speedwise. It checks MIFARE Classic tags sector keys against a dictionary file with keys",
+                  "This is a speed improvement of the checkkeys method.\n"
+                  "It can be used to check a MIFARE Classic tag's sector keys against a dictionary file (.dic) with keys.",
                   "hf mf fchk --mini -k FFFFFFFFFFFF              --> Key recovery against MIFARE Mini\n"
                   "hf mf fchk --1k -k FFFFFFFFFFFF                --> Key recovery against MIFARE Classic 1k\n"
                   "hf mf fchk --2k -k FFFFFFFFFFFF                --> Key recovery against MIFARE 2k\n"
                   "hf mf fchk --4k -k FFFFFFFFFFFF                --> Key recovery against MIFARE 4k\n"
+                  "hf mf chk --1k -k FFFFFFFFFFFF -k A0A1A2A3A4A5 --> Check all sectors, A&B keys against MIFARE Classic 1k\n"
```

Comment:
`fchk`

Reply:
Just catching up.
Line (approx.) 116 
PrintAndLogEx(FAILED, "This is expected to happen in 25% of all cases. Trying again with a different reader nonce...");

Don't we need %% to escape the % ?